org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/customservice/synapse/mediator/tryout/server/MIServer.java

```diff
@@ -94,16 +109,39 @@ public synchronized void startServer() {
         }
         try {
             serverProcess = startServerProcess();
-            handleKeystorePassword();
+
+            BufferedReader reader = new BufferedReader(
+                    new InputStreamReader(serverProcess.getInputStream(), StandardCharsets.UTF_8));
+
+            handleKeystorePassword(reader);
+
```

Comment:
Shall we remove unnecessary line spaces?

Reply:
fixed in [Remove empty lines](https://github.com/wso2/mi-language-server/pull/430/commits/e9f697efc00f04923435ba3cfb47e6212306e8b6)